src/main/scala/exercises/02_Fibonacci.scala

```diff
@@ -1,13 +1,19 @@
 package exercises
 
 object Fibonacci {
-  
   /**
    * Implement the classic Fibonacci sequence, where
    *    fib(1) = 1
    *    fib(2) = 1
    *    fib(n) = fib(n-1) + fib(n-2)
    */
-  def fib(n: Int): Int = 0
+  def fib(n: Int): Int = quick_fib(n)._1
+
+  private[this] def quick_fib(n: Int): (Int, Int) = if (n <= 1)
+    (1, 1)
```

Comment:
I didn't know that you can make things private this way.

I think it'd be more readable if `if (n <= 1)` was on a separate line. You still wouldn't need the curly braces for the function body since it is one if else block.